trento/migration/systemd-install.md

```diff
@@ -487,12 +435,78 @@ This is a basic guide for creating a self-signed certificate for use with Trento
 
    If there are issues with the configuration, the output will indicate what needs to be adjusted.
 
-1. Reload Nginx to apply changes:
+1. If SSL certificates were modified or added, reload NGINX to apply changes.
 
    ```bash
    systemctl reload nginx
    ```
 
+## Prepare SSL certificate for NGINX
+
+Create or provide a certificate for [NGINX](https://nginx.org/en/) to enable SSL for Trento.
+This is a basic guide for creating a self-signed certificate for use with Trento. You may use your own certificate. For detailed instructions, consult the [OpenSSL documentation](https://www.openssl.org/docs/man1.0.2/man5/x509v3_config.html).
+
+### Option 1: Creating a Self-Signed Certificate
+
+1.  Generate a self signed certificate:
+
+    > Note: Remember to adjust `subjectAltName = DNS:trento.example.com` by replacing `trento.example.com` with your own domain and change the value `5` to the number of days for which you need the certificate to be valid. For example, `-days 365` for one year.
+
+    ```bash
+    openssl req -newkey rsa:2048 --nodes -keyout trento.key -x509 -days 5 -out trento.crt -addext "subjectAltName = DNS:trento.example.com"
+    ```
+
+1.  Copy the generated trento.key to a location accessible by NGINX:
+    ```bash
+    cp trento.key /etc/ssl/private/trento.key
+    ```
+
+1.  Create a directory for the generated trento.crt which is accessible by NGINX:
+   
+    ```bash
+    mkdir -p /etc/nginx/ssl/certs/
+    ```
+
+1.  Copy the generated trento.crt to the previously created folder:
+   
+    ```bash
+    cp trento.crt /etc/nginx/ssl/certs/trento.crt
+    ```
+
+1. Reload NGINX to apply changes:
+
+   ```bash
+   systemctl reload nginx
+   ```
+
+### Option 2: Create a signed certificate with Let's Encrypt using PackageHub repository
+
+> **Note:** Change repository if you use a Service Pack other than SP5. For example: [SLE15 SP3: `SUSEConnect --product PackageHub/15.3/x86_64`,SLE15 SP4: `SUSEConnect --product PackageHub/15.4/x86_64`].
+> Users should assess the suitability of these packages based on their own risk tolerance and support needs.
+
```

Comment:
What are "these packages"? If you want to warn the reader about enabling PackageHub, then you have to be more direct. Something like: Use packages in PackageHub at your own risk.

Reply:
@dmpop yes, that is precisely what we are trying to say. I like your suggestion.